src/lib/columns/Columns.js

```diff
@@ -72,7 +88,7 @@ class Columns extends Component {
           'rct-vl' +
           (firstOfType ? ' rct-vl-first' : '') +
           (minUnit === 'day' || minUnit === 'hour' || minUnit === 'minute'
-            ? ` rct-day-${time.day()} `
+            ? ` rct-day-${time.getDate()} `
```

Comment:
same point here as before, I am not sure that we want a classe with the date instead of the day.

Reply:
[improve utility fcts](https://github.com/Optiways/react-calendar-timeline/pull/1/commits/a30c37ad347b34dfa524e63e2c61d389184123de)